http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -47,6 +47,12 @@ WELDER_OWNER_EMAIL="${13:-dummy}"
 WELDER_STAGING_BUCKET="${14:-dummy}"
 WELDER_STAGING_STORAGE_CONTAINER_RESOURCE_ID="${15:-dummy}"
 
+# Envs for Azure
+WORKSPACE_ID="${10:-dummy}"
```

Comment:
I think these args need to start at `${16-dummy}` and go to `${19-dummy}`

Reply:
Ah, I see. Maybe could just rename the above variable to `WORKSPACE_ID` and reuse it?

Reply:
Sounds good!